module/VuFind/src/VuFind/Cache/Manager.php

```diff
@@ -410,4 +416,49 @@ protected function createFileCache($cacheName, $dirName, $overrideOpts = [])
             ],
         ];
     }
+
+    /**
+     * Create an in-memory cache
+     *
+     * @param array $storageConfig See Storage in RateLimiter.yaml
+     *
+     * @return StorageInterface
+     */
+    public function createInMemoryCache(array $storageConfig): StorageInterface
+    {
+        $adapter = $storageConfig['adapter'] ?? 'memcached';
+
+        // Use cache manager for "VuFind" cache (only for testing purposes):
```

Comment:
I think this comment is a bit confusing, especially now that it has been moved to a new context. Maybe this would be better:

```suggestion
        // The 'vufind' adapter uses a standard file-based cache to simulate an in-memory cache.
        // This is intended for TESTING PURPOSES ONLY, since it allows us to test related functionality
        // without setting up a real in-memory data store. It should not be used for any other purpose.
```

Reply:
Yup much better